src/main/java/homework_2/pyramid_printer/PyramidPrinter.java

```diff
@@ -0,0 +1,43 @@
+package homework_2.pyramid_printer;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class PyramidPrinter {
+
+    private final String ERROR = "Only 1 non-negative integer is allowed as passed parameter";
+
+    public void run() {
+        System.out.println("Please, input a number:");
+        int input = consoleReader();
+        if (negativeInput(input)) {
+            System.out.println(ERROR);
+        } else if (input == 0) {
+            System.out.println();
+        } else {
+            printResult(input);
+        }
```

Comment:
```suggestion
        if (negativeInput(input)) {
            System.out.println(ERROR);
        } else {
            printResult(input);
        }
```
and 
`    private void printResult(int input) {`
`        String output = "";`
`        for (int i = 0; i < input; i++) {`
 `           output += "x";`
`        }`
`        System.out.println(output);`
`    }`
    will it work this way?